CompuCell3D/core/Demos/BookChapterDemos_ComputationalMethodsInCellBiology/VascularTumor/README.txt

```diff
@@ -0,0 +1,7 @@
+This demo is incorrect and pending fixes. The bug is that all cells generate in the same area. 
```

Comment:
I might need help fixing this one.

Reply:
This demo looks OK. The model is 3D and you can't see the other cells unless you display the cell field in 3D.
![Untitled](https://github.com/CompuCell3D/CompuCell3D/assets/22944062/509c35c9-a04d-4ee2-a698-06f8231612f1)

Might want to change the _settings.sqlite file so that there a four windows: the cell field plus the three diffusion fields. And have the cell field shown in 3D.

Reply:
> This demo looks OK. The model is 3D and you can't see the other cells unless you display the cell field in 3D. ![Untitled](https://private-user-images.githubusercontent.com/22944062/346632475-509c35c9-a04d-4ee2-a698-06f8231612f1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0NTQ0MDQsIm5iZiI6MTcyMDQ1NDEwNCwicGF0aCI6Ii8yMjk0NDA2Mi8zNDY2MzI0NzUtNTA5YzM1YzktYTA0ZC00ZWUyLWE2OTgtMDZmODIzMTYxMmYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDE1NTUwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJjODE5MzRmNWNhMDFlZGU0Nzk2NzhmYTNjZDc5YTc4OWQwNGMwNjBjOTZjYjYwYWQzNDA1NmM4OTc5NGE1NzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5r5wEk_7eSdcAxTL_t4IJNhaBtQ1A9bcppTdGCzpEN0)
> 
> Might want to change the _settings.sqlite file so that there a four windows: the cell field plus the three diffusion fields. And have the cell field shown in 3D.

Also might change the description so it is clear to the user that the model is 3D and needs to be displayed in 3D.

Reply:
I'm glad that the solution was so simple.